sdk/ai/ai-projects/samples-dev/agents/agents_basics_tracing_console.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+
+import { context } from "@opentelemetry/api";
+import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import { createAzureSdkInstrumentation } from "@azure/opentelemetry-instrumentation-azure-sdk";
+import {
+    ConsoleSpanExporter,
+    NodeTracerProvider,
+    SimpleSpanProcessor,
+} from "@opentelemetry/sdk-trace-node";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+
+registerInstrumentations({
+    instrumentations: [createAzureSdkInstrumentation()],
+});
+
+import { AIProjectsClient } from "@azure/ai-projects"
+import { DefaultAzureCredential } from "@azure/identity";
+
+const connectionString = process.env["AZURE_AI_PROJECTS_CONNECTION_STRING"] || "<endpoint>>;<subscription>;<resource group>;<project>";
```

Comment:
Samples should ideally show how to auth using more secure methods like entra ID and such - is that capability available and can we update the sample to show it?

Reply:
discussed offline about connection string and auth.